resources/ory/charts/oathkeeper/templates/deployment-controller.yaml

```diff
@@ -18,6 +18,8 @@ metadata:
       {{- toYaml . | nindent 4 }}
     {{- end }}
 spec:
+  seccompProfile:
```

Comment:
I think this is not valid here ([reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/#DeploymentSpec)) and should be moved to `template.spec.securityContext`.